src/app/core/uri-retriever.service.spec.ts

```diff
@@ -1,188 +1,204 @@
+//       httpTestingController.expectOne(mockResponse.resourceUris.metrics).flush({saves: 10, downloads: 10});
+//       httpTestingController.expectOne(mockResponse.resourceUris.parents).flush([]);
+//     });
+
+//     it('should get the Learning Object by id with all of the available resources', done => {
+//       const mockParams = {
+//         id: '12345678901234567890',
+//       };
+
+//       uriRetrieverService.getLearningObject(
+//         mockParams,
+//         ['children', 'outcomes', 'materials', 'metrics', 'parents', 'ratings']
+//       ).subscribe(object => {
+//         expect(object.name).toBe(mockResponse.name);
+//         expect(object.children).toEqual([]);
+//         expect(object.outcomes).toEqual([]);
+//         expect(object.materials).toEqual([]);
+//         expect(object.metrics).toEqual({saves: 0, downloads: 0});
+//         expect(object.parents).toEqual([]);
+//         expect(object.ratings).toBeUndefined();
+//         done();
+//       });
+
+//       const uri = `${environment.apiURL}/learning-objects/${mockParams.id}`;
+//       httpTestingController.expectOne(uri).flush(mockResponse);
+//       httpTestingController.expectOne(mockResponse.resourceUris.children).flush([]);
+//       httpTestingController.expectOne(mockResponse.resourceUris.materials).flush([]);
+//       httpTestingController.expectOne(mockResponse.resourceUris.metrics).flush({saves: 0, downloads: 0});
+//       httpTestingController.expectOne(mockResponse.resourceUris.outcomes).flush([]);
+//       httpTestingController.expectOne(mockResponse.resourceUris.parents).flush([]);
+//       httpTestingController.expectOne(mockResponse.resourceUris.ratings).flush(null);
+//     });
+
+//     it('should get the Learning Object by id of the learning object ', done => {
+//       const mockParams = {
+//         id: '12345678901234567890',
+//       };
+
+//       uriRetrieverService.getLearningObject(
+//         mockParams,
+//         ['children', 'materials', 'parents']
+//       ).subscribe(object => {
+//         expect(object.name).toBe(mockResponse.name);
+//         expect(object.children).toEqual([]);
+//         expect(object.materials).toEqual([]);
+//         expect(object.parents).toEqual([]);
+//         done();
+//       });
+
+//       const uri = `${environment.apiURL}/learning-objects/${mockParams.id}`;
+//       httpTestingController.expectOne(uri).flush(mockResponse);
+//       httpTestingController.expectOne(mockResponse.resourceUris.children).flush([]);
+//       httpTestingController.expectOne(mockResponse.resourceUris.materials).flush([]);
+//       httpTestingController.expectOne(mockResponse.resourceUris.parents).flush([]);
+//     });
+
+//     it('should throw and error with insufficient learning object identifiers', done => {
+//       expect(() => uriRetrieverService.getLearningObject({})).toThrowError();
+//       done();
+//     });
+//   });
+// });
```

Comment:
@pzalep1 should we just remove these tests?

Reply:
Yes, these tests don't really do much